Sources/Fuzzilli/Lifting/JSExpressions.swift

```diff
@@ -15,6 +15,7 @@
 // JavaScript expressions. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence
 public let Identifier           = ExpressionType(precedence: 20,                        inline: .always)
 public let Literal              = ExpressionType(precedence: 20,                        inline: .always)
+public let RegExpLiteral        = ExpressionType(precedence: 20,                        inline: .singleUseOnly)
```

Comment:
Maybe add a comment here or in the JavaScriptLifter.swift to explain why we handle them differently? I guess something like 
```swift 
// RegExp are objects, and so for example for the FuzzIL program 
//     v1 <- CreateRegExp
//     Compare v1, v1
// there is a difference between 
//    let a = /a/;
//    a === a;  // (true)
// and 
//     /a/ === /a/;  // (false)
// the former being the correct JavaScript equivalent.
```
or so?